core/server.go

```diff
@@ -368,6 +373,9 @@ func (m *RpcServer) ControlEnvironment(cxt context.Context, req *pb.ControlEnvir
 		Id: env.Id().String(),
 		State: env.CurrentState(),
 		CurrentRunNumber: env.GetCurrentRunNumber(),
+		StartOfTransition: sot.Format(time.RFC3339Nano),
+		EndOfTransition: eot.Format(time.RFC3339Nano),
+		TransitionDuration: strconv.FormatFloat(td.Seconds(), 'f', 6, 64),
```

Comment:
I think we should look at keeping a standards across the gRPC files;

Right now I can see that AliECS returns dates as string in the following format: `2022-03-15T21:02:20+01:00`. I took the example from `GetEnvironment` and `GetEnvironments` methods. ( `string createdWhen = 2;` ). Maybe this was chosen as this format so that it goes well with `coconut`?

I would be a fan of starting using normal timestamps from now on and we bring the already existing string ones to this format. This would be what you suggested @teo `int64`